pages/dkp/kommander/2.2/operations/access-control/index.md

```diff
@@ -66,24 +66,6 @@ subjects:
     name: oidc:engineering
 ```
 
-## Types of Access Control Objects
```

Comment:
I think this is all still relevant and should not be removed.

Reply:
Rodney, it is still relevant but it's already explained in the section above. This seemed to me a duplicate section. Will you verify?